agent/agents/mysql/slowlog/parser/parser_test.go

```diff
@@ -136,6 +136,85 @@ func TestParserSpecial(t *testing.T) {
 		assert.Equal(t, expect, actual)
 	})
 
+	t.Run("parseTime", func(t *testing.T) {
+		t.Parallel()
+
+		r, err := NewSimpleFileReader(filepath.Join("testdata", "slow023.log"))
+		require.NoError(t, err)
+		defer func() {
+			assert.NoError(t, r.Close())
+		}()
+
+		opts := log.Options{
+			DefaultLocation: time.UTC,
+		}
+		p := NewSlowLogParser(r, opts)
+
+		p.parseTime("Time: 180214 16:18:07")
+		assert.Equal(t, p.event.Ts, time.Date(2018, time.February, 14, 16, 18, 7, 0, time.UTC))
+	})
+
+	t.Run("parseUser", func(t *testing.T) {
+		t.Parallel()
+
+		r, err := NewSimpleFileReader(filepath.Join("testdata", "slow023.log"))
+		require.NoError(t, err)
+		defer func() {
+			assert.NoError(t, r.Close())
+		}()
+
+		opts := log.Options{
+			DefaultLocation: time.UTC,
+		}
+		p := NewSlowLogParser(r, opts)
+		p.parseUser("User@Host: bookblogs[bookblogs] @ localhost []  Id: 56601")
+		assert.Equal(t, p.event.User, "bookblogs")
+		assert.Equal(t, p.event.Host, "localhost")
+	})
+
+	t.Run("parseMetrics", func(t *testing.T) {
+		t.Parallel()
+
+		r, err := NewSimpleFileReader(filepath.Join("testdata", "slow023.log"))
```

Comment:
same question is applicable to all tests, it is not clear why "slow023.log" is used.

Reply:
ok, what should be used?